geomstats/geometry/hypersphere.py

```diff
@@ -597,6 +597,33 @@ def dist(self, point_a, point_b):
 
         return dist
 
+    def dist_pairwise(self, point):
```

Comment:
This should be a method of the class RiemannianMetric, since it will be the same code for all the child classes.

Reply:
True! thanks